monitoring/src/main/scala/akkaviz/events/LightSnapshot.scala

```diff
@@ -36,9 +39,17 @@ case class LightSnapshot(
       copy(liveActors = liveActors - ref)
     case CurrentActorState(ref, _) if ref.isUserActor =>
       copy(liveActors = liveActors + ref)
-    case Instantiated(ref, _) if ref.isUserActor =>
-      copy(liveActors = liveActors + ref)
+    case Instantiated(ref, actor) if ref.isUserActor =>
+      copy(liveActors = liveActors + ref, classes = classes.updated(ref, actor.getClass.getName))
```

Comment:
I think we need to filter out (this is general problem) all things which are not actor, eg. ActorSystem from the list